pyftpdlib/handlers.py

```diff
@@ -3471,6 +3492,14 @@ def get_ssl_context(cls):
                 if not cls.keyfile:
                     cls.keyfile = cls.certfile
                 cls.ssl_context.use_privatekey_file(cls.keyfile)
+                if cls.client_certfile is not None:
+                    cls.ssl_context.set_verify(VERIFY_PEER |
+                                               VERIFY_FAIL_IF_NO_PEER_CERT |
+                                               VERIFY_CLIENT_ONCE,
+                                               cls.verify_certs_callback)
+                    cls.ssl_context.load_verify_locations(cls.client_certfile)
+                    cls.ssl_context.set_session_cache_mode(SESS_CACHE_OFF)
+                    cls.ssl_options = cls.ssl_options | OP_NO_TICKET
```

Comment:
It would be nice to add a comment explaining what SESS_CACHE_OFF and OP_NO_TICKET do.